### DISCUSSION TITLE
In the lava stdp implementation are A+ and A- flipped from what they should be?

### DISCUSSION THREAD

**Discussion by Michaeljurado42** [lava-nc/lava#483] [Q&A]
Specifically, I am looking at this declaration of the STDP learning rule inside the tutorial:

![image](https://user-images.githubusercontent.com/18356610/201694534-59e9ad67-ab28-468b-8792-50f5d62dac93.png)

In the tutorial they link a wiki page resource showing classic STDP hebbian learning:
![image](https://user-images.githubusercontent.com/18356610/201694866-0ef781f5-4fd4-44ab-85aa-a8a425853f9d.png).
Using a negative A+ does lead to weight increases according to this equation. 

I notice that in the stdp tutorial they also flip the order of A+ and A- so the graph looks correct:
![image](https://user-images.githubusercontent.com/18356610/201695667-643a8833-fb85-4c86-8175-dd19242dcc11.png)
. In this example on_post_stdp is A-

I am not sure if this is an issue or an intentional choice which is why I made it a discussion post.

**Comment by weidel-p** [comment]
Hi @Michaeljurado24,

thanks for bringing that to our attention. I double checked and you are right indeed, the usage of `Aplus` and `Aminus` is flipped and this is **not** intended. We will correct that (@gkarray). It would be great if you could add an issue about this anyway, just to make sure it's not forgotten.

**Reply by Michaeljurado42** [reply]
I just made an[ issue](https://github.com/lava-nc/lava/issues/484). Thanks for the reply!
